api/app/signals/apps/api/v1/serializers/category.py

```diff
@@ -108,3 +109,59 @@ def update(self, instance, validated_data):
             instance.refresh_from_db()
 
         return super(PrivateCategorySerializer, self).update(instance, validated_data)
+
+
+class PrivateCategoryHistoryHalSerializer(serializers.ModelSerializer):
+    identifier = serializers.SerializerMethodField()
+    what = serializers.SerializerMethodField()
+    action = serializers.ReadOnlyField(source='get_action_display')
+    description = serializers.SerializerMethodField()
+    _category = serializers.IntegerField(source='object_id', read_only=True)
+
+    class Meta:
+        model = Log
+        fields = (
+            'identifier',
+            'when',
+            'what',
+            'action',
+            'description',
+            'who',
+            '_category',
+        )
+
+    def get_identifier(self, log):
+        return f'{log.get_action_display().upper()}_CATEGORY_{log.id}'
+
+    def get_what(self, log):
+        return log.get_action_display().upper()
+
+    def get_description(self, log):
+        description = []
+        for key, value in log.data.items():
+            if key == 'name':
+                title = f'Naam wijziging'
+            elif key == 'description':
+                title = f'Omschrijvings wijziging'
+            elif key == 'slo':
+                title = f'Service level agreement wijziging'
+
+                sla_list = ServiceLevelObjective.objects.filter(pk__in=value)
+                sla_value = []
+                for sla in sla_list:
+                    if sla.use_calendar_days:
+                        sla_value.append(f'{sla.n_days} weekdagen')
+                    else:
+                        sla_value.append(f'{sla.n_days} werkdagen')
+                value = '\n'.join(sla_value)
+            elif key == 'is_active':
+                title = f'Status wijziging'
+                value = 'Actief' if value else 'Inactief'
+            else:
+                continue  # We do not show other tracked values, so on to the next one
+
+            description.append({
+                'title': title,
+                'text': value
+            })
```

Comment:
These are also new semantics. Description was just a string (in the `Signal` history case). Analog here would be to make an array of history entries --- but that will not work given the fact that only one `Log` entry is created when several fields are touched.